services/diversity/src/persistence/richness_nos.js

```diff
@@ -19,22 +19,20 @@ module.exports = (
    *
    * @returns {Object[]} Number of total inferred and observed species.
    */
-  findTotalNumberOfSpecies: async (areaType, areaId) => {
-    try {
-      return db('richness_nos as rn')
-        .select(
-          db.raw('coalesce(rn.total_inf, 0) as inferred'),
-          db.raw('coalesce(rn.total_obs, 0) as observed'),
-          db.raw('coalesce(rnr.total_obs, 0) as region_observed'),
-          db.raw('coalesce(rnr.total_inf, 0) as region_inferred'),
-        )
-        .leftJoin('richness_nos_regions as rnr', 'rn.id_region', 'rnr.id_region')
-        .where({ 'rn.geofence_type': areaTypeKeys(areaType), 'rn.geofence_id': areaId });
-    } catch (e) {
-      logger.error(e.stack || e.Error || e.message || e);
-      throw new Error('Error getting data');
-    }
-  },
+  findTotalNumberOfSpecies: async (areaType, areaId) => (
```

Comment:
you should delete de `async` here too

Reply:
Should I delete async in findSCIHFLayerInEA of SCIHF,js ? as in the richness_nos we do not use async.

Reply:
also richness_nos neither use try and catch for the raw querys. Is it correct to remove try-catch raw ?